server/src/main/java/org/elasticsearch/repositories/blobstore/BlobStoreRepository.java

```diff
@@ -543,11 +613,21 @@ public long getRestoreThrottleTimeInNanos() {
         return restoreRateLimitingTimeInNanos.count();
     }
 
+    protected void verificationThreadCheck() {
```

Comment:
we're not only checking the verification thread here, but all operations that are accessing the blobstore. Can you rename this method?